composer.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "litespeedtech/lscache_wp",
+	"require-dev": {
+		"squizlabs/php_codesniffer": "*",
+		"phpcompatibility/php-compatibility": "*"
+	},
+	"prefer-stable": true,
+	"scripts": {
+		"post-install-cmd": [
+			"\"vendor/bin/phpcs\" --config-set installed_paths vendor/phpcompatibility/php-compatibility"
+		],
+		"post-update-cmd": [
+			"\"vendor/bin/phpcs\" --config-set installed_paths vendor/phpcompatibility/php-compatibility"
+		],
+		"sniff-check": [
+			"./vendor/bin/phpcs -p . --standard=./phpcs.xml.dist"
+		]
+	}
+}
```

Comment:
There should be a newline at end of file.

Reply:
It get removed by Prettier. Should I add it manually?

Reply:
No, that's fine.